grader_service/convert/converters/base.py

```diff
@@ -264,6 +264,8 @@ def matches_allowed_patterns(file_path):
             """
             Check if a file matches any of the allowed glob patterns.
             """
+            if file_path.endswith(".ipynb"):
+                return False
```

Comment:
Please add a comment explaining why this exception is needed here (you can probably use the comment from `grader-labextension`)